bundler/lib/bundler/templates/newgem/.circleci/config.yml.tt

```diff
@@ -0,0 +1,13 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: ruby:<%= RUBY_VERSION %>
+    steps:
+      - checkout
+      - run:
+          name: Run tests
+          command: |
+            gem install bundler -v <%= Bundler::Version %>
+            bundle install
+            bundle exec rake test
```

Comment:
Sorry for going back and forth here

Since now `--test` and `--ci` flags are completely independent, let's not make the CI configuration "test specific". Let's name the default step "Run the default task", and run the `bundle exec rake` command, whatever that is. Same kind of thing would apply for all CI configurations.

Reply:
Generic templates are good! 3/4 CI templates were updated.